src/components/AssetsSearch/AssetsSearch.test.jsx

```diff
@@ -10,36 +11,24 @@ const defaultProps = {
   courseDetails: {},
 };
 
-let wrapper;
 
 describe('<AssetsSearch />', () => {
   describe('renders', () => {
+    let wrapper;
```

Comment:
Out of curiosity, what was the reasoning around moving this down? Just want to make sure that I understand.

Reply:
The muscle memory: I tend to put variable declarations into the closest possible enclosing scope whenever writing JS.

Effects:
- helps clarify what scope something is needed for
- when adding references to the variable to wider scopes, forces you to acknowledge that that could be a confusing change by forcing you to move the declaration
- (no longer a real problem with ES6 modules) helps avoid accidentally globally-scoped variables